### ISSUE TITLE
Tests are not detected if comment exists after #[test] attribute

### ISSUE THREAD

**Issue by MunifTanjim** [rouge8/neotest-rust#37]
```rs
#[test]
/// a word
fn test_a_word() {
    process_reverse_case("robot", "tobor");
}
```
